lib/client-api/src/story_store.ts

```diff
@@ -341,7 +345,7 @@ export default class StoryStore {
       getOriginal,
       storyFn,
 
-      parameters,
+      parameters: { ...storyParameters, argTypes },
```

Comment:
This means `parameters.argTypes` always exists, even if it is `{}`. I wasn't sure about that @shilman. WDYT?

Reply:
Hmm 🤔 seems like it would be useful to have null argTypes, but i don’t think the current code depends on that. We can revisit later if needed